src/style/style_layer/custom_style_layer.js

```diff
@@ -7,7 +7,7 @@ import assert from 'assert';
 import type {ValidationErrors} from '../validate_style.js';
 import type {ProjectionSpecification} from '../../style-spec/types.js';
 
-type CustomRenderMethod = (gl: WebGLRenderingContext, matrix: Array<number>, projection: ?ProjectionSpecification, projectionToMercatorMatrix: ?Array<number>, projectionToMercatorTransition: ?number) => void;
+type CustomRenderMethod = (gl: WebGLRenderingContext, matrix: Array<number>, projection: ?ProjectionSpecification, projectionToMercatorMatrix: ?Array<number>, projectionToMercatorTransition: ?number, centerInMercator: ?Array<number>, pixelsPerMeterRatio: ?number) => void;
```

Comment:
Adding more parameter gives me another reason to point out this suggestion: https://github.com/mapbox/mapbox-gl-js/pull/12182#discussion_r1084569681 to clean this interface before we publish it publicly.

Or, if we could eliminate most of these parameters per above comment altogether that would be even more ideal to reduce a bit of the complexity for the user; e.g. remove the need to pass along `globeToMercMatrix`, `transition`, `centerInMercator`, and `pixelsPerMeterRatio`.